app/logic/searchUsers.py

```diff
@@ -11,15 +11,21 @@ def searchUsers(query):
     lastName = " ".join(splitSearch[1:]) +"%"
 
     if len(splitSearch) == 1: #search for first or last name
-        results = User.select().where(User.isStudent & (User.firstName ** firstName | User.lastName ** firstName))
+        if category =="searchCategory":
+            results = User.select().where( User.isFaculty & (User.firstName ** firstName | User.lastName ** firstName))
+        else:
+            results = User.select().where(User.isStudent & (User.firstName ** firstName | User.lastName ** firstName))
         for participant in results:
             if participant not in resultsDict:
-                resultsDict[participant.username]=model_to_dict(participant)
+                resultsDict[participant.username]= model_to_dict(participant)
     else:
-        for searchTerm in splitSearch: #searching for specified first and last name
+        for searchTerm in splitSearch: #searching for both first and last names
             if len(searchTerm) > 1:
                 searchTerm += "%"
-                results = User.select().where(User.isStudent & User.firstName ** firstName & User.lastName ** lastName)
+                if category =="searchCategory":
+                    results = User.select().where( User.isFaculty & (User.firstName ** firstName | User.lastName ** firstName))
+                else:
+                    results = User.select().where(User.isStudent & (User.firstName ** firstName | User.lastName ** firstName))
```

Comment:
You changed these queries so they are the same as the other queries. Searching for "Brian Rams" would not work anymore.

Reply:
Yes, Brian Rams would not show up because he is neither staff/faculty nor student but he is a CELTS admin. Therefore, none of the situations applies to him.